docs/source/what_is_jupyter.md

```diff
@@ -0,0 +1,412 @@
+  experience
+
+- JupyterLab offers a more customizable, feature-rich, tabbed multi-notebook
+  editing environment, and has additional tools like a customizable layout
+  and system console (a common tool used by programmers)
+
+- And more...read about additional notebook interfaces here (LINK)!
+
+A professor, for instance, might use JupyterLab for their daily work as a data
+analyst, and Jupyter Notebook while teaching to provide a simpler, more focused
+view of the notebook's content to their students.
+
+### The notebook format
+
+- [nbformat.readthedocs.io](https://nbformat.readthedocs.io)
+
+Jupyter uses the .ipynb format to store your notebook files (where the data
+and outputs of your notebook reside for long term storage). The .ipynb format
+is interpreted and modified by the nbformat software library.
+
+TODO: features, more info, basic exmaplanation of how it works/structure
+
+### JupyterHub
+
+- [jupyterhub.readthedocs.io](https://jupyterhub.readthedocs.io)
+
+JupyterHub provides a multi-user management system where many different people
+can log in and use their own isolated notebook editor program and environment.
+
+With JupyterHub, editing and running notebook files is still performed by an
+editor program like Lab or Notebook. The Hub is responsible for authenticating
+users and providing them their corresponding Lab/Notebook instance connection.
+
+Jupyter-Hub can be set up in different system configurations:
+
+- In a single computer
+- In a cluster of computers
+- In containers in the cloud
+
+JupyterHub is quite flexible for many different multi-user scenarios.
+
+### IPython
+
+- [IPython]([ipython.readthedocs.io/interactive/tutorial]: https://ipython.readthedocs.io/en/stable/interactive/tutorial.html)
+
+A long time ago, a precursor to Jupyter was created, called ipython (see
+the [History of Jupyter][] to learn more about that). It provided some of
+the same fast, advanced REPL features that Jupyter still has today, and it
+is the default Python kernel bundled with JupyterLab and Notebook. Some of
+the features ipython provides:
+
+- Foobar
+- Bizbaz
+- WikRakRum
+- interactive shell: [ipython.readthedocs.io/interactive/tutorial][]
+- magic commands: [ipython.readthedocs.io/interactive/magics][]
+
+For more details on IPython features check [`docs.jupyter.org > Projects > IPython`](https://docs.jupyter.org/en/latest/projects/ipython_projects.html).
+
+
+
+### Interactive Buttons, Sliders and more with ipywidgets
+
+- [Jupyter Widgets (AKA ipywidgets)](https://ipywidgets.readthedocs.io/en/latest/)
+
+Jupyter Widgets (also called ipywidgets) provides interactive interface
+elements that you can add directly into your notebooks.
+
+- Note: The subproject and high-level references to the widgets were renamed to
+Jupyter-Widgets, but Jupyter-Widgets and IPyWidgets are the very same thing. To
+not break compatibility with older code, the software library is still called
+`ipywidgets`.
```

Comment:
These may be a bit too detailed for a basic introduction.

I feel like, for a user just getting started with Jupyter, they should learn first about the notebook concept and 1 or _maybe_ 2 apps (Notebook and Lab). Additional documentation can shed light on other projects.